tests/CommandLineMock.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DBrekelmans\BrowserDriverInstaller\Tests;
+
+use DBrekelmans\BrowserDriverInstaller\CommandLine\CommandLineEnvironment;
+use RuntimeException;
+use function Safe\sprintf;
+
+class CommandLineMock implements CommandLineEnvironment
```

Comment:
I don't think this class is necessary as we can mock the interface in the test classes like we had before. I try to stay away from mock classes as a mock could have different needs in different tests, so I want to keep them specific to each test.

Reply:
OK I put back previous implementation. I wanted to try a different way of mocking the class ;-)